### PR TITLE
Sparkle dark

### PR DESCRIPTION
**Pull request by Duncid**
- Polishing Radio and other components for Darkmode

### CODE REVIEW

File: sparkle/src/components/Page.tsx [hunk 1]
```diff
@@ -134,6 +135,59 @@ const gapSizes = {
...
+        />
+      );
+    default:
+      return (
+        <Page.Vertical
+          children={children}
+          sizing={sizing}
+          align={align}
+          gap={gap}
+        />
```
Comment (PopDaph) [inline sparkle/src/components/Page.tsx:180]: As direction is typed, we should never be there. We can return null; throw an error, or keep as is but remove the `case "vertical":` that is duplicated?

File: sparkle/src/components/RadioButton.tsx [hunk 2]
```diff
@@ -38,37 +48,36 @@ export function RadioButton({
 }: RadioButtonProps) {
   return (
     <div className={classNames("s-flex s-gap-3", className || "")}>
-      {choices.map((choice) => {
-        if (value === choice.value) console.log("selected", choice.value);
```
Comment (PopDaph) [inline sparkle/src/components/RadioButton.tsx:42]: Thanks for removing this 🙈

**Review by PopDaph — DISMISSED** [review]

File: sparkle/src/components/Page.tsx (hunk 1 above)
Comment (Duncid) [reply sparkle/src/components/Page.tsx:180]: Ok! Thought having a default was mandatory. Changing.

Comment (PopDaph) [reply sparkle/src/components/Page.tsx]: @Duncid having a default is mandatory I suggested to return null instead, or keep the default and remove case vertical !

**Pull request merged**